src/includes/product.ts

```diff
@@ -16,6 +16,14 @@ export interface Product extends Record<string, any> {
   variants?: ProductVariants;
 }
 
+/**
+ * The type of a function used to transform record data into products.
+ *
+ * @typeparam T The type of the product into which the Record will be
```

Comment:
We're not referring to the `Record` type, so "record" is the correct capitalization.

```suggestion
 * @typeparam T The type of the product into which the record will be
```